test/trento_web/plugs/api_redirector_test.exs

```diff
@@ -72,12 +74,36 @@ defmodule TrentoWeb.Plugs.ApiRedirectorTest do
              } == resp
     end
 
-    test "should redirect to the correct path when the route is recognized with the latest version",
+    test "should redirect to the newest path when the route with the newest version is available",
+         %{conn: conn} do
+      conn =
+        conn
+        |> Map.put(:path_info, ["api", "test"])
+        |> ApiRedirector.call(available_api_versions: ["v2", "v1"], router: FoundRouter)
+
+      assert 307 == conn.status
+
+      location_header = get_resp_header(conn, "location")
+
+      assert ["/api/v2/test"] == location_header
+    end
+
+    test "should redirect to the next newest path when the route when it is the first available version",
```

Comment:
I am not getting this, maybe a typo.
I guess you meant something like: should redirect to the latest available version of an endpoint :thinking:

Reply:
yes, typo...

Reply:
Fixed